docs/contributions-guide/decision-log.md

```diff
@@ -14,3 +14,9 @@ We've decided to use http://beghp.github.io/ domain to deploy each branch there
 
 ### Cypress (13 March 2020)
 We've decided that [Cypress](https://github.com/cypress-io/cypress) is better than [Hermione](https://github.com/gemini-testing/hermione) (Hermione predecessor is [Gemini](https://github.com/gemini-testing/gemini)) because Hermione required setting a very large tolerance in order to pass both locally and in CI. Moreover, Cypress is much more popular.
+
+### Recoil (14 July 2020)
+We've decided that [recoil](https://recoiljs.org/) will be used for managing state of the react-carousel component.
```

Comment:
could you also write why recoil is better than redux for react-carousel?

Reply:
done